Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM ubuntu:trusty
```

Comment:
Does this install the latest stable Node.js version (currently 4.4.4)? Should we use the official Node.js Docker image [here](https://hub.docker.com/_/node/)?